antora-ui-camel/src/css/blog.css

```diff
@@ -223,3 +223,13 @@ article.blog p {
 .blog .page-item.active a {
   color: var(--color-white);
 }
+
+@media screen and (max-width: 1023px) {
+  article.blog:first-child {
+    margin-top: 5rem;
+  }
+
+  .blog.list aside {
+    margin-top: 4.5rem;
```

Comment:
If we move the whole `--body-top` then we don't need to reposition the individual elements, like the category menu here.